src/controllers/collateralController.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config();
+const Web3 = require('web3');
+const Tx = require('ethereumjs-tx');
+import {
+  AssetCollateral,
+  GAS_PRICE,
+} from '../constants';
+
+const ADDRESS = process.env.ADDRESS;
+const ADDRESS_PRIVATE_KEY = Buffer.from(process.env.ADDRESS_PRIVATE_KEY, 'hex');
```

Comment:
Do we have a way to check whether this `ropsten` key has funds to perform these operations?

Reply:
Not right now. But also people aren't really withdrawing collaterals since it takes us a while to generate revenue for the existing assets (which in turn will allow people to withdraw their collateral in percentages). So we essentially have a throttling system in place. Plus our main address (0xBB) still has almost 100 Million MYB (Ropsten). Once the address that handles collateral needs some MYB I'll fill it up. That and the move to the SDK will remove this code.